upi/vsphere/vm/nm-keyfile.tmpl

```diff
@@ -0,0 +1,13 @@
+[connection]
+id=Static ethernet connnection 1
+uuid=${uuid}
+type=802-3-ethernet
+interface-name=ens192
+autoconnect=true
+
+[ipv4]
+method=manual
+dns=${dns}
+dns-search=${cluster_domain}
+addresses=${ip_address}/24
```

Comment:
Hmm.. why the fixed network range?

Reply:
it seems to be the default. I haven't seen any examples without it, and we've been using it with this range in OKD as well: https://github.com/openshift/installer/blob/fcos/upi/vsphere/machine/ignition.tf#L41 (that's still TF 0.11x). Happy to change this if needed.

Reply:
I'll try to look, but how did we decide the /24, and what does it mean to set it to that value.

Also it seem like we have machine_cidr, how does that relate to this value..